scripts/assign-colors.py

```diff
@@ -15,6 +25,7 @@
     parser.add_argument('--color-schemes', type=str, required=True, help="input color schemes file")
     parser.add_argument('--metadata', type=str, help="if provided, restrict colors to only those found in metadata")
     parser.add_argument('--clade-node-data', type=str, help="if provided, restrict to only those clades found in tree")
+    parser.add_argument('--clade-recency', type=int, nargs='?', const=None, help="if provided, restrict to clades found in tree within X months of present")
```

Comment:
If you opted for the approach I described above of building a `params.clade_recency_argument` in the workflow, you could drop the `nargs='?', const=None,` arguments here, since that's already the default behavior for optional arguments.

Reply:
Awesome! Resolved in 497cb9ce63bbf92dc400994463b367dfac7df6bd.